Unable to build

Hi!

So I followed steps described in [installation tutorial](https://github.com/nebulasio/wiki/blob/master/tutorials/%5BEnglish%5D%20Nebulas%20101%20-%2001%20Installation.md), however I am unable to run make build.

When I run `make build`, it gives me following response:

<img width="1400" alt="screen shot 2018-01-07 at 20 49 15" src="https://user-images.githubusercontent.com/15366622/34653455-44994cd4-f3ec-11e7-879a-a6287452bfa9.png">


Any ideas how to solve this issue? Sorry if it is a dumb question, I don't have a whole lot experience with GO.

Thanks!

Hi,

I have the same issue, when I execute make build I have this errors: 
cd cmd/neb; go build -ldflags "-X main.version=0.6.0 -X main.commit=a5ead6d389afae736e2b876020e5a9acc4fdc1ec -X main.branch=develop -X main.compileAt=`date +%s`" -o ../../neb
# github.com/go-nebulas/cmd/neb
./crashclient.go:41:3: undefined: logging.CLog
./crashclient.go:61:3: undefined: logging.CLog
./crashclient.go:79:7: conf.CrashReportUrl undefined (type *nebletpb.AppConfig has no field or method CrashReportUrl)
./crashclient.go:83:3: undefined: logging.CLog
./crashclient.go:91:3: undefined: logging.CLog
./crashclient.go:110:3: undefined: logging.CLog
./crashclient.go:117:2: undefined: logging.CLog
./main.go:85:2: undefined: logging.Init
./main.go:88:59: n.Config().App.CrashReportUrl undefined (type *nebletpb.AppConfig has no field or method CrashReportUrl)
./main.go:123:10: conf.App.Version undefined (type *nebletpb.AppConfig has no field or method Version)
./main.go:123:10: too many errors
make: *** [build] Error 2

--- a/tutorials/[English] Nebulas 101 - 01 Installation.md
+++ b/tutorials/[English] Nebulas 101 - 01 Installation.md
@@ -73,6 +73,9 @@ If you need the full commit history, clone all to local:
 ```
 git clone https://github.com/nebulasio/go-nebulas.git
 ```
+Note: If you opt for the second option, don't forget to checkout to `master` branch or to commit `ed37bea` (v0.5.0.), otherwise this tutorial might not work for you.
+
+
 Since Go must compile in `$GOPATH`,  Nebulas code must be in `/src/github.com/nebulasio/go-nebulas` under `$GOPATH`。
 
 Note: so make sure you create a src folder and put nebulas inside it.
@@ -474,5 +477,3 @@ Use the above configuration file to start the node:
 ```
 
 NAS testing network can be obtained from the official website for testing. Distribution Address: [https://testnet.nebulas.io/claim/](https://testnet.nebulas.io/claim/).
-
-

